python/singa/autograd.py

```diff
@@ -283,7 +439,8 @@ def forward(self, x):
         Returns:
             a new CTensor whose element y = x if x >= 0; otherwise 0;
         '''
-        if training:
+        self.param={'name':'LeakyRelu','x':x,'alpha':0.0}
```

Comment:
1. indent alignment
2. why do you need to add an attribute `param`?

Reply:
1. I need to save names of models' layer
2. and the data used for the current layer.